package.json

```diff
@@ -21,6 +23,8 @@
   "dependencies": {
     "axios": "^0.22.0",
     "form-data": "^4.0.0",
+    "node-cron": "^3.0.0",
+    "pm2": "^5.1.2",
```

Comment:
don't make it dependencies, pm2 should be global right?

Reply:
Doesn't have to. It also works in this way. Just cutting the process for people to install it manually globally.